pkgs/development/python-modules/pytest-testinfra/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, ansible-core
+, paramiko
+, pytest
+, pytestCheckHook
+, pytest-xdist
+, pywinrm
+, salt
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-testinfra";
+  version = "8.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-m0CCi1j7esK/8pzBRlk0rfQ08Q3VoQj2BTXe5SZgpj0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  nativeCheckInputs = [
+    ansible-core
+    paramiko
+    pytest
+    pytestCheckHook
+    pytest-xdist
+    pywinrm
+    salt
+  ];
+
+  preCheck = ''
+    export HOME=$(pwd)
+  '';
+
+  pytestFlagsArray = [
+    "--disable-pytest-warnings"
+  ];
```

Comment:
Those should be disabled by default. is some config file enabling them like project.toml, setup.cfg or pytest.ini? If so we should patch them rather.